slack-base/src/main/java/com/hubspot/slack/client/models/blocks/ActionsIF.java

```diff
@@ -0,0 +1,26 @@
+package com.hubspot.slack.client.models.blocks;
+
+import java.util.List;
+
+import org.immutables.value.Value;
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.blocks.elements.BlockElement;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ActionsIF extends Block {
+  String TYPE = "actions";
+
+  @Override
+  @Value.Derived
+  default String getType() {
+    return TYPE;
+  }
+
+  List<BlockElement> getElements();
```

Comment:
Since this is the singleton field that folks need to set, I think we want to add `@Paramater` so that Immutables generates `Actions.of(...)` so you can just create such a block from a list of elements.

Reply:
Oh, I didn't know this was a thing!!  I'll go through each Immutable I've added and add `@Parameter` to all the required fields.

Reply:
(I've always used this in cases where there's a single field that can be used to construct it, so that you can build it with an `of` call)